### ISSUE TITLE
Developer dashboard - Logs

### ISSUE THREAD

**Issue by thSoft** [FTSRG/incqueryd#81]
[ ] Log to HDFS per application
[ ] Separate log per Akka actor / Spark job
[ ] Auto-refresh
[ ] Simultaneous text search

**Comment by thSoft** [comment]
I enabled YARN log aggregation in a11247da61a3141cbbbd251cf38df1f957b0ce8a. Logs are aggregated on HDFS in /tmp/logs (by default), but unfortunately an application's logs are only aggregated _after the application finished_. So it does not help us because the actor system and the Spark applications run forever.

**Comment by thSoft** [comment]
Alternatives for the implementation:
* implement a HDFS log appender
* reuse Hadoop's web interface - depends on #88
* integrate with [Flume](flume.apache.org) - seems overkill

**Comment by thSoft** [comment]
Log aggregation to HDFS implemented in 7736689b50ad640613299cd3f2d8abc646c77695.

**Comment by thSoft** [comment]
Please:
* [ ] merge into master
* [ ] test on EC2
* [ ] document running the dashboard on the wiki

**Comment by thSoft** [comment]
HDFS-FUSE is unreliable, Hadoop's logging mechanism should be intercepted instead by overriding its `log4j.properties` to use a HDFS appender.

**Comment by thSoft** [comment]
Currently, logs are aggregated to HDFS which is mounted properly, but all of them is empty.
